common/src/main/java/dev/isxander/yacl3/gui/utils/ItemRegistryHelper.java

```diff
@@ -0,0 +1,103 @@
+package dev.isxander.yacl3.gui.utils;
+
+
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.Items;
+
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+public class ItemRegistryHelper {
+
+	private static Item getItemFromNameImpl(String value) {
```

Comment:
I think the standard is to suffix with `0` rather than `Impl`. Additionally this should be near the bottom of the class since it's private.